tesseract.js/tesseract.js.d.ts

```diff
@@ -0,0 +1,149 @@
+// Type definitions for tesseract.js
+// Project: https://github.com/naptha/tesseract.js
+// Definitions by: York Yao <https://github.com/plantain-00/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="../node/node.d.ts" />
+
+declare module Tesseract {
+    type ImageLike = string | HTMLImageElement | HTMLCanvasElement | HTMLVideoElement
+        | CanvasRenderingContext2D | File | Blob | ImageData | Buffer;
+    type Progress = {
```

Comment:
sure, it's done now.
Though I think it's better to improve the error message from inside typescript.